### PR TITLE
feat: #135 scroll functionnality and tabs

### PR DESCRIPTION
**Pull request by kjacqueschu**
- Scroll functionality working (reset on filter change and tabs change)
- Tabs design
- Add links between tabs when clicking nb donors or nb files

image
image

### CODE REVIEW

File: src/packages/@cqdg/pages/FileRepository/RepoCasesTable/RepoCasesTable.js [hunk 1]
```diff
@@ -28,6 +24,11 @@ import './CasesTable.css';
 export default compose(
   setDisplayName('RepoCasesTablePresentation'),
   connect(state => ({ tableColumns: state.tableColumns.cases })),
+  withPropsOnChange(['variables'], ({ variables: { cases_size } }) => {
```
Comment (francisl) [inline src/packages/@cqdg/pages/FileRepository/RepoCasesTable/RepoCasesTable.js]: no need for return here

File: src/packages/@cqdg/pages/FileRepository/RepoCasesTable/RepoCasesTable.js [hunk 2]
```diff
@@ -48,13 +50,7 @@ export default compose(
...
     return (
       <div className="cases-table">
-        <Row
-          style={{
-            backgroundColor: 'white',
-            padding: '1rem',
-            justifyContent: 'space-between',
-          }}
-          >
+        <Row className="cases-actions">
```
Comment (francisl) [inline src/packages/@cqdg/pages/FileRepository/RepoCasesTable/RepoCasesTable.js]: Row and Column can be replaced by `<StackLayout />` and `<StackLayout vertical />`

File: src/packages/@cqdg/pages/FileRepository/RepoCasesTable/RepoCasesTable.model.js [hunk 3]
```diff
@@ -181,7 +142,24 @@ const casesTableModel = [
     downloadable: true,
     hidden: true,
     th: () => <Th>Number of Files</Th>,
-    td: ({ node }) => <Td>{node.files && node.files.hits ? node.files.hits.total : 0}</Td>,
+    td: ({ node }) => {
```
Comment (francisl) [inline src/packages/@cqdg/pages/FileRepository/RepoCasesTable/RepoCasesTable.model.js]: no need to return here. We should place it in the linter/prettier like clin or kidsfirst

**Review by francisl — COMMENTED** [review]
Looking good, two small comments

File: src/packages/@cqdg/pages/FileRepository/RepoCasesTable/RepoCasesTable.model.js (hunk 3 above)
Comment (kjacqueschu) [reply src/packages/@cqdg/pages/FileRepository/RepoCasesTable/RepoCasesTable.model.js]: I console logged before something and forgot to change it back :p

**Review by francisl — APPROVED** [review]
🚢

**Pull request merged**